package.json

```diff
@@ -44,7 +44,7 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-es2015": "^6.24.1",
-    "bcrypt": "^1.0.3",
+    "bcrypt": "^3.0.0",
```

Comment:
So the issue with `bcrypt` was that when I initially started this project locally, I was running node version 8.6.1. I upgraded to node version 10.10.0 and on re-installing dependencies, bcrypt installation would always fail because the node version used in building it had changed. My solution was to remove it from dev-dependencies and delete packagelock.json (because the version there was also 1.0.3). Installing a fresh copy of bcrypt did the trick and I think that's probably the problem on Heroku

Reply:
Makes sense. In that case, this PR will help resolve that issue (hopefully).